packages/pluggableWidgets/sidebar-web/src/Sidebar.editorConfig.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    hidePropertiesIn,
+    Properties,
+    StructurePreviewProps,
+    transformGroupsIntoTabs
+} from "@mendix/piw-utils-internal";
+
+import { SidebarPreviewProps } from "../typings/SidebarProps";
+
+export function getProperties(
+    values: SidebarPreviewProps,
+    defaultProperties: Properties,
+    platform: "web" | "desktop"
+): Properties {
+    if (values.toggleMode !== "startExpandedShrink" && values.toggleMode !== "startCollapsedShrink") {
```

Comment:
Honestly think that negation makes this harder to follow because you need to be aware of all the different enumeration values. I suggest changing both of these conditions to highlight the ones that are relevant rather than irrelevant

```suggestion
    if (values.toggleMode === "none" && values.toggleMode === "push" && values.toggleMode === "slideOver") {
```

Reply:
I think you mean `||` rather than `&&`? 😅

Reply:
Sharp @tamelbrun!